gno.land/pkg/gnoland/app.go

```diff
@@ -146,3 +182,23 @@ func EndBlocker(vmk vm.VMKeeperI) func(ctx sdk.Context, req abci.RequestEndBlock
 		return abci.ResponseEndBlock{}
 	}
 }
+
+func guessGnoRootDir() string {
```

Comment:
This helper will need to be moved later, but I've a refactor plan in mind, so feel free to commit it here and we'll unify after.